src/main/anovos/feature_recommender/feature_recommendation.py

```diff
@@ -0,0 +1,406 @@
+                                             columns=['Input Attribute Name', 'Recommended Feature Name',
+                                                      'Recommended Feature Description', 'Feature Similarity Score',
+                                                      'Industry',
+                                                      'Usecase', 'Source'])
+                else:
+                    df_append = pd.DataFrame(
+                        [[df_user[desc_column].iloc[i], 'Null', 'Null', 'Null', 'Null', 'Null', 'Null']],
+                        columns=['Input Attribute Name', 'Recommended Feature Name',
+                                 'Recommended Feature Description', 'Feature Similarity Score',
+                                 'Industry',
+                                 'Usecase', 'Source'])
+            else:
+                if float(single_score) >= threshold:
+                    df_append = pd.DataFrame([[df_user[name_column].iloc[i], df_user[desc_column].iloc[i],
+                                               df_rec_fr['Feature Name'].iloc[int(idx)],
+                                               df_rec_fr['Feature Description'].iloc[int(idx)],
+                                               "%.4f" % (cos_scores[idx]),
+                                               df_rec_fr['Industry'].iloc[int(idx)],
+                                               df_rec_fr['Usecase'].iloc[int(idx)],
+                                               df_rec_fr['Source'].iloc[int(idx)]]],
+                                             columns=['Input Attribute Name', 'Input Attribute Description',
+                                                      'Recommended Feature Name',
+                                                      'Recommended Feature Description', 'Feature Similarity Score',
+                                                      'Industry',
+                                                      'Usecase', 'Source'])
+                else:
+                    df_append = pd.DataFrame(
+                        [[df_user[name_column].iloc[i], df_user[desc_column].iloc[i], 'Null', 'Null', 'Null', 'Null',
+                          'Null', 'Null']],
+                        columns=['Input Attribute Name', 'Input Attribute Description', 'Recommended Feature Name',
+                                 'Recommended Feature Description', 'Feature Similarity Score',
+                                 'Industry',
+                                 'Usecase', 'Source'])
+            df_out = df_out.append(df_append, ignore_index=True)
+    return df_out
+
+
+def find_attr_by_relevance(df, building_corpus, name_column=None, desc_column=None, threshold=0.3):
+    """
+    :param df: Input DataFrame - Attribute Dictionary
+    :param building_corpus: Input Feature Description (list)
+    :param name_column: Input column name for Attribute Name in Input DataFrame (string). Default is None.
```

Comment:
Input, column name of Attribute Name in Input DataFrame (string). Default is None.